### DISCUSSION TITLE
api方式接入，多环境密钥的问题

### DISCUSSION THREAD

**Discussion by whutWuXiaofeng** [apolloconfig/apollo#4915] [Q&A]
非spring的java项目，api方式接入，可以通过apollo-env.properties中配置多密钥接入吗，或者支持其他拆分配置文件的方式吗

**Comment by nobodyiam** [comment]
多密钥接入是啥？

**Reply by whutWuXiaofeng** [reply]
不同的环境，配置了不同的密钥，我看官方文档apollo-env.properties支持配置多meta地址，然后指定env，是否支持配置各环境的密钥呢

**Reply by nobodyiam** [reply]
密钥是分环境的

[6.2 配置访问密钥](https://www.apolloconfig.com/#/zh/usage/apollo-user-guide?id=_62-%e9%85%8d%e7%bd%ae%e8%ae%bf%e9%97%ae%e5%af%86%e9%92%a5)
